Extension/.vscode/tasks.json

```diff
@@ -166,6 +166,34 @@
                     }
                 }
             ]
+        },
+        {
+            "label": "Compile:Watch (unit-tests)",
+            "type": "shell",
+            "options": {
+                "cwd": "${workspaceFolder}"
+            },
+            "command": "node_modules/.bin/tsc --build ${workspaceFolder}/test.tsconfig.json --watch",
```

Comment:
The other tasks use yarn plus script from the package.json. Should we follow that pattern here as well?

Reply:
Sure!